python/Ganga/Runtime/bootstrap.py

```diff
@@ -181,6 +181,10 @@ def manualExportToGPI(my_interface=None):
     from Ganga.GPIDev.Lib.Registry.JobRegistry import jobSlice
     exportToInterface(my_interface, "jobSlice", jobSlice, "Functions")
 
+    """Needed for the JobTime if we plan to use the interface to eval the XML as well"""
+    import datetime
+    exportToInterface(my_interface, "datetime", datetime, 'Classes')
```

Comment:
Is there any particular file where you feel this applies?
If we can get the performance improvements without adding more code I'm happy but I've specifically stripped out a lot of changes and gunk in this PR to get to the minimal changes which fix several performance issues which I've had a lot of contact from users over.